wiki/content/deploy/index.md

```diff
@@ -1933,6 +1935,10 @@ ending in .rdf, .rdf.gz, .json, and .json.gz will be loaded.
 `--format`: Specify file format (rdf or json) instead of getting it from
 filenames. This is useful if you need to define a strict format manually.
 
+`-x, --store_xids`: Generate a xid edge for each node. It will store the XIDs (The identifier / Blank-nodes) in an attribute named `xid` in the entity itself. It is useful if you gonna use [External IDs](/mutations#external-ids).
```

Comment:
This [PR](https://github.com/dgraph-io/dgraph/pull/5148) removes `-x` flag from bulk loader to avoid discrepancy between live loader and bulk loader.